### ISSUE TITLE
Breaking change: need to start layout with basic_table()

### ISSUE THREAD

**Issue by anajens** [Roche/rtables#251]
Testing with the latest pre-release (10c1f187c452051bb3762293dea381a3487e2487), there was a breaking change introduced where layouts must start with `basic_table` and the code below produces an error.

Is this intentional? We're making downstream changes now across NEST but given that it's a breaking change we need to also communicate this to users.
  
```r
library(rtables)

split_cols_by(lyt = NULL, "ARM") %>%
  analyze(c("SEX", "AGE")) %>%
  build_table(ex_adsl)

```

```
Error in h(simpleError(msg, call)) : 
  error in evaluating the argument 'obj' in selecting a method for function 'last_rowsplit': nope. can't add a col split to that (NULL). contact the maintaner.
```

**Comment by gmbecker** [comment]
It was intentional but I will take it back out and deprecate instead.

~G

**Comment by anajens** [comment]
Ok that sounds good. Please let me know when you are ready as I would like to test first against `tern` on your feature branch. If it's all good then you can merge to `pre-release`.

**Comment by gmbecker** [comment]
Pushed to gabe_tabletree_work.

Its now "supported" but is deprecated so will throw a warning.  Will change
from deprecated to defunct right after this CRAN release in main branch.

~G

**Comment by gmbecker** [comment]
Fix in e8e1a24 this is now deprecated but wills till work (with a warning). It will be removed fully in a future release.

**Comment by anajens** [comment]
Thanks @gmbecker, everything looks OK in tern. Feel free to merge to `pre-release`.
